tmail-backend/tmail-third-party/openpaas/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.linagora.tmail</groupId>
+        <artifactId>tmail-third-party</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>tmail-openpaas</artifactId>
+    <name>Twake Mail :: Third Party :: OpenPaaS</name>
+    <description>OpenPaaS integration for Twake Mail</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
Once again dependecny ordering

Reply:
They are sorted alphabetically by their artifact name. If project dependencies go first, that wouldn't be the case anymore. Should I group project dependencies in one group and other dependencies in their own group and sort each alphabetically?

Reply:
Sort alphabeticalyh on thre groupid then artifact name.

$project first, then $james then the rest